.github/workflows/cd-traing-pipeline.yml

```diff
@@ -29,11 +33,16 @@ jobs:
     
     - uses: actions/setup-python@v4
       with:
-        python-version: '3.8'
+        python-version: '3.9'
         
     - name: 'Install required Python packages'
       run: |
         pip install -r requirements.txt
+
+    - name: 'Install cpp Protocol Buffer'
+      run: |
+        export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=cpp
+        sudo pip install --upgrade --force-reinstall protobuf==3.17.3
```

Comment:
Why this one needed?

Reply:
ah sorry about this one.
at the time, GitHub Action complained about something about protocol buffer, but seems like working OK now. will remove this one.